packages/katello/rubygem-foreman_inventory_upload/rubygem-foreman_inventory_upload.spec

```diff
@@ -1,4 +1,4 @@
-# Generated from foreman_inventory_upload-1.0.0.beta1.gem by gem2rpm -*- rpm-spec -*-
+# Generated from foreman_inventory_upload-1.0.0.beta3.gem by gem2rpm -*- rpm-spec -*-
```

Comment:
You can drop this line. We don't really use this and at this point it's mostly noise. Note the next one (`# template:`) is important.